deployment/dockerdeploy/deployer.go

```diff
@@ -1163,6 +1163,94 @@ func (d *Deployer) UnpauseNode(ctx context.Context, clusterID string, nodeID str
 	return nil
 }
 
+func (d *Deployer) FailOverNode(ctx context.Context, clusterID string, nodeID string, failOverType string, rebalance bool, allowUnsafe bool) error {
+	node, err := d.getNode(ctx, clusterID, nodeID)
+	if err != nil {
+		return errors.Wrap(err, "failed to get cluster info")
+	}
+	controller, err := d.getController(ctx, clusterID)
+	if err != nil {
+		return errors.Wrap(err, "failed to get controller for cluster")
+	}
+
+	otp := "ns_1@" + node.IPAddress
```

Comment:
I think there are a few cases where the OTP of the node will not always match exactly like this.  If you look at our add/remove node handling, you should find the OTP of the node like it does rather than assuming its format.

Reply:
Done.